src/Components/Aspire.Azure.Messaging.EventHubs/Aspire.Azure.Messaging.EventHubs.csproj

```diff
@@ -10,7 +10,7 @@
     <PackageIconFullPath>$(SharedDir)AzureEventHubs_256x.png</PackageIconFullPath>
     <NoWarn>$(NoWarn);SYSLIB1100;SYSLIB1101</NoWarn>
     <!-- This package hasn't shipped stable, so package validation runs against previously shipped version of it. -->
-    <PackageValidationBaselineVersion>8.0.1-preview.8.24267.1</PackageValidationBaselineVersion>
+    <PackageValidationBaselineVersion>8.1.0-preview.1.24373.2</PackageValidationBaselineVersion>
```

Comment:
Why does this one need this, but the others that haven't shipped stable don't?

Reply:
All of the ones that haven't shipped stable should need it. If not I'm expecting build will fail.